src/assets/scss/application.scss

```diff
@@ -91,3 +91,11 @@ $govuk-assets-path: "/public/";
   margin-top: 0; 
   margin-bottom: 20px;
 }
+
+.landingPageDetailBullets {
```

Comment:
Do you reckon we could use a government design system class here instead of creating our own?

Reply:
Have changed to govuk-!-margin-left-3 👍🏻